includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -611,6 +611,7 @@ public function scripts() {
 				'needs_payer_phone' => 'required' === get_option( 'woocommerce_checkout_phone_field', 'required' ),
 			],
 			'button'          => [
+				// TODO: migrate values.
```

Comment:
I'll do this separately, since there already seem to be enough changes in here :D